src/core/logical_layers/utils/logicalLayerFabric.ts

```diff
@@ -200,8 +202,15 @@ export function createLogicalLayerAtom(
                 state.settings?.name || state.id || 'Disaster Ninja map layer'
```

Comment:
app name (Disaster Ninja) here and below should be taken from config

Reply:
or just should not be

Reply:
There's the same issue with geojson download. 
Should it be changed for both?

> app name (Disaster Ninja) here and below should be taken from config `configRepo.get().name`